src/test/java/edu/oakland/test/database/DatabaseCommManagerStub.java

```diff
@@ -0,0 +1,31 @@
+package edu.oakland.test.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabaseCommManager;
+
+public class DatabaseCommManagerStub implements DatabaseCommManager {
+  int rfid = 10;
+  LocationDataPoint ldp = null;
+
```

Comment:
Can we add a way to set the RFID number? One way to do it is via constructor (as suggested), or you could also do this via a setter.
```suggestion

  public DatabaseCommManagerStub(int rfid) {
    this.rfid = rfid;
  }

```